soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -32,7 +32,7 @@ u8 generated;
 
 const std::string Randomizer::getItemMessageTableID = "Randomizer";
 const std::string Randomizer::hintMessageTableID = "RandomizerHints";
-const std::string Randomizer::scrubMessageTableID = "RandomizerScrubs";
+const std::string Randomizer::merchantMessageTableID = "RandomizerScrubs";
```

Comment:
now that it's not just scrubs can we change this to `"RandomizerMerchants"`?

Reply:
Oh yeah, just missed this one. Honestly thought might change these to enum'd IDs instead of strings in a future revision of custom_message stuff and re-use the mod-index stuff from get-item-rework. Not 100% sure though since rando has multiple message tables to itself.

Reply:
Resolved in 53531c4481123e448f731f01d5152ac0bd5cf049